tools/topology/topology2/sof-ace-tplg/CMakeLists.txt

```diff
@@ -21,6 +21,9 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-rt711-4ch.bin,DEEPBUFFER_FW_DMA_MS
 
 "cavs-rt5682\;sof-mtl-max98357a-rt5682\;PLATFORM=mtl,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-max98357a-rt5682.bin,DEEPBUFFER_FW_DMA_MS=10"
+
+"cavs-rt5682-ssp0spk\;sof-mtl-max98357a-rt5682-ssp0spk\;PLATFORM=mtl,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,NUM_HDMIS=4,\
+PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-max98357a-rt5682.bin,DEEPBUFFER_FW_DMA_MS=10"
```

Comment:
this looks very similar to the previous topology, with the exception of the SSP being swapped.

Can we not reuse the same topology and macros to avoid duplication?

Also not clear why we have the same NHLT blobs if the SSPs are different.

And last why do we have HDMI supported and not on line 22?